transfers/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('', views.login_view, name='login'),
```

Comment:
I don't think we want to define an entire new login view. What we always did, and we should do, is let the built-in `django.contrib.auth.view`s do their work. They will authenticate the users more neatly for us.